src/main.cpp

```diff
@@ -5,6 +5,8 @@
 #include "G4RunManager.hh"
 #include "G4UImanager.hh"
 #include "QGSP_BERT_HP.hh"
+#include "LBE.hh"
```

Comment:
Not related to the crystal material - right?

Reply:
Done

Reply:
Moved to another MR: https://github.com/bwynneHEP/SimplePetScanner/pull/3